# Mattias Schertell

#### _Universitetsadjunkt/Studierektor - [Blekinge Tekniska H√∂gskola](https://www.bth.se/)_ 

#### Kortfattat: 
Jag √§r universitetsadjunkt, studierektor och bitr. programansvarig p√• institutionen Teknik och estetik (DITE). 
De senaste √•ren har jag varit delaktig i utvecklingen av en distansutbildningen Design av digital upplevelser f√∂r l√§rande. Jag undervisar inom det programmet inom framf√∂rallt tekniska √§mnen s√•som programmering och webbproduktion. 

Jag har varit anst√§lld p√• BTH sedan 2004 och i huvudsak undervisat inom medieteknik och datavatenskapliga √§mnen, framf√∂rallt kopplat till webb; front-end och back-end, objektorienterad programmering och agila metoder. Det √§r dessa √§mnen jag intresserad av och brinner f√∂r.
Parallellt med undervisning s√• √§r jag studierektor och representant i olika centrala administrativa r√•d och grupper. Jag har ocks√• varit programansvarig och medlem i ett antal olika projekt som utvecklare .

Jag har en magisterexamen i Gestaltning i digitala medier.



#### Tekniska f√§rdigheter & kunskaper:

- **Programmeringsspr√•k:** _javascript, python, java, C#, php; erfarenhet av Go, bash, Clojure_
- **√ñvrigt:** _relationsdatabaser, NoSql, git, UML, Linux (t.ex. command line, ssh m.m.); erfarenhet av exempelvis Docker, Apache Kafka ._
- **Metoder och processer**: _Scrum, Kanban, design patterns ._

---


## üéì Utbildning
- **Magisterexamen i Gestaltning i digitala medier**	| <a href="https://www.bth.se/" target="_blank">BTH</a> (_AUG 2003 - JUN 2004_)	 			        		
- **Teknologie kandidatexamen i Medieteknik** | <a href="https://www.bth.se/" target="_blank">BTH</a> (_AUG 2000 - JUN 2003_)

## üî¨ Vidareutbilnding
**Medietekniks handledningsprocesser MEA037 BTH (_2006_)**
- Den h√§r kursen motsvarar "H√∂gskolenpedagogik" om totalt 15hp

**Handledning i h√∂gre utbildning (_2014_)**
- Kurs f√∂r personal inom BTH f√∂r att utveckla sin pedagogiska skicklighet.
- Kursansvarig √Öse Nygren.

**Klart ledarskap (_2021_)**
- Ledarutvecklingsprogram f√∂r att utveckla sina f√§rdigheter i att leda sig sj√§lva och andra genom formellt och informellt ledarskap..
- Kursansvarig Camilla R√ºden.

## üíº Arbetslivserfarenhet 
**Universitetsajunkt @ <a href="https://www.bth.se/" target="_blank">BTH</a> (_JUN 2004 - p√•g√•ende_)**
- **Kursansvar**, **examinator**, **undervisning** och **handledning** i kurser inom webbutveckling och webbprogrammering (fron-end och back-end), objektorienterad programmering, systemutveckling och agila metoder samt i flera kurser inom programmen Digitala och immersiva upplevelser, Digital bildproduktion, Digital ljudproduktion.
- **Programansvarig** - inom programmet Medieteknik med inriktning mot iteraktiva system ansvarade jag f√∂r och utvecklade kurser med inneh√•ll inom front-end- och back-end-utveckling. Detta resulterade i programmet Webbutveckling som gjorde startade 2009. Jag tog √∂ver programmansvaret 2010-2018. Programmet gjorde ett namnbyte till Interaktion med webbteknologier, d√§r jag var programansvarig 2016-2019.
- **Studierektor** sedan 2017. Inkluderar arbete med institutionens tj√§nsteplanering, budget, verksamhetsm√•l och kursplanering. Inf√∂rde team-baserat arbete vilket effektiviserade och f√∂renklade planering och genomf√∂rande av kurser.
- **Studieadministrativt r√•d** - jag √§r medlem i SA-r√•det. Tillsammans med andra studierektorer, utbildningsst√∂d, programst√∂d och studentk√•ren √§r man delaktig i sp√∂rsm√•l och beslut som r√∂r centrala studieadminstrativa √§renden.
- **Utbildningsr√•d (UR)** - jag har varit medlem i utbildningsr√•det som l√§rarrepresentant fakulteten Datavetenskaper sedan 2021. R√•det behandlar √§renden som ber√∂r utbildningar (tex utbildningsplaner) och √§mnseomr√•den och f√∂rbereder beslut f√∂r Utbildningsutskottet.
- **IT referensgrupp** jag √§r representant f√∂r min institution i denna grupp sedan 2016. Gruppens syfte √§r att informera om och vara r√•dg√∂rande kring central infrastruktur och mjukvara inom BTH.

**Back-end utvecklare @ <a href="https://www.managerzone.com/" target="_blank">ManagerZone</a> (_2007_)**
- Som back-end utvecklare p√• ManagerZone bestod mina arbetsuppgifter av att utveckla funktionalitet f√∂r KPI (Key Perfomance Indicators). Detta innefattade att ta fram komponenter f√∂r m√§ta och identifiera vilka delar av spelet som var (mest) inkomstbringande (ROI).
Jag ledde ocks√• ett arbete f√∂r en migrering fr√•n versionshanteringssystemet CVS till SVN. 
Jag ingick i ett SCRUM-team och arbetade i huvudsak med javscript, php och mysql. F√∂retagets huvudkontor var baserat i Karlskrona och under 2007 blev man uppk√∂pta av <a href="https://en.wikipedia.org/wiki/DoubleClick" target="_blank">Doubleclick</a> som i sin tur blev uppk√∂pta av Google √•ret efter.

## üíª Programming and data analysis skills
- Proficient in full stack development of **R packages** using modular coding practices.
- Created **production-grade Shiny web applications** for interactive data analysis and visualization and; expertise in dependency management tools like Golem for scalable deployment.
- **Machine learning model** building for image classification/segmentation tasks; trained CNNs and other deep learning architectures in R, Python (PyCharm).
- **Multivariate data analysis** of large-scale omics datasets including genomics, phenomics and metabolomics using cutting-edge bioinformatics tools.
- Experience with analysis of **next-generation sequencing** data including quality control, read mapping, variant calling, expression quantification, metagenomic profiling, and associated statistical analysis using standard workflows in R and Python.
- **Advanced visualization** for multi-dimensional biological data through Circos, ggtree, ggtreeextra, Cytoscape and other platforms.

**Additional Skills:**
  
- Git/GitHub for version control and collaborative coding.
- High performance computing on clusters for scalable data analysis.
- Bioconductor for analyzing genomic/transcriptomic experiments.
- Workflow automation to enhance reproducibility and to faster workflow. 

## üå± R Packages developed
### ‚úÖ PBGeno
<a href="https://github.com/Allan-gitrepos/pbgeno" target="_blank">GitHub</a>

Developed pbgeno, an R package to streamline data analysis workflows for plant breeders. The package provides functions for calculating genetic distances, clustering genotypes, estimating diversity statistics, creating publication-quality visualizations, and automating routine tasks. Key features include calculating genetic distance matrices, structure-based clustering, polymorphism quantification, and converting proprietary marker genotypes into standardized formats for genome-wide association mapping. 

### ‚úÖ PBPerfect
**PBPerfect** <a href="https://allanbiotools.shinyapps.io/pbperfect/" target="_blank">(Visit Page)</a> is a interactive web tool enabling reproducible multivariate analysis with visualization of phenotypic and genotypic data. It features basic statistics, experimental designs, SSR workflows, multivariate analysis, mating designs, and dynamic graphics with outputs exported as publication-standard tables and graphics requiring no further formatting.

### ‚úÖ PBMLT: Plant Breeding Multilocation Trail Data Analysis Software

**PBMLT**<a href="https://allanbiotools1.shinyapps.io/PBMLT/" target="_blank">(Visit Page)</a> is a comprehensive and user-friendly platform that provides plant breeders with an all-in-one solution for analyzing multi-environment trial data through:

- Powerful Analysis of Variance (ANOVA) to explore significance of variation across locations and treatments.
- Additive Main Effects and Multiplicative Interaction (AMMI) analysis for in-depth genotype-environment interaction studies.
- Calculation of essential AMMI-based stability indices like ASV, ASTAB, ASI, MASI, SIPC, ZA for identifying adaptable lines.
- Evaluation of overall productivity using metrics such as mean AVAMGE for high-yielding genotype selection.
- Scaled stability measures like SSIASTAB and ASI_SSI for ranking lines based on trait stability.
- Interactive visualizations including Biplots, GGE plots, WASS plots for straightforward interpretation of complex data.
- Centralized platform integrating meta-analysis, statistical analytics, and genotype-environment interaction analysis.

### ‚úÖ PBlinkagemap

**PBlinkagemap** <a href="https://allanbiotools1.shinyapps.io/PBLinkagemap/" target="_blank">(Visit Page)</a> enables easy creation of linkage maps and identification of associated quantitative trait loci (QTLs) from genomic and phenotypic datasets. 

It allows users to:
- Import chromosome, marker, map distance and trait score data
- Interactively explore results on linkage maps
- Visualize QTL locations and effects
By handling computationally intensive linkage analysis and mapping behind the scenes, PBlinkagemap makes it simple for users to go from datasets to QTL discovery through an intuitive interface.

### ‚úÖ PB-GWAS üß¨

**PB-GWAS** <a href="https://allanbiotools1.shinyapps.io/PBgwas/" target="_blank">(Visit Page)</a> makes powerful genome-wide association studies accessible through an easy-to-use web app üë©‚Äçüíª

Key features:
üì• File upload in 4 clicks  
üèÉ‚Äç‚ôÇÔ∏è One-click GWAS launch  
‚öôÔ∏è Adjust parameters via sidebar   
üìà Interactive result plotting  
üìÑ Full PDF report downloading

By eliminating coding barriers, PB-GWAS allows both new and advanced users to leverage GAPIT workflows with no programming expertise required!. Whether you want to map simple or complex traits, PB-GWAS provides the automated analysis to accelerate discoveries üî¨

### ‚úÖ PBHaploMineR üß¨

**PBHaploMineR** <a href="https://github.com/Allan-gitrepos/PBhaplominerR" target="_blank">(Visit Page)</a> provides a toolkit to streamline pangenome haplotype mining and comparison from next-generation sequencing data. This R package aims to make large-scale haplotype analysis efficient and accessible for species with reference pangenomes.

**Key Features:**
* **Sequence Import** - Functions to import raw reads from multiple platforms and store in standardized schema
* **Haplotype Calling** - Optimized algorithms for pangenome-wide haplotype calling, incorporating structural variation
* **HapViz** - Interactive visualization system to explore and compare haplotypes in context of pangenome structure
* **HapCompare** - Statistically compare haplotypes between groups of samples/accessions and identify associated genomic signatures
* **Parallelization** - Built-in parallelization to scale analyses across HPC infrastructure

PBHaploMineR is still under development and testing. ETA for first stable version is Q1 2024.

## üé§ Workshop and Conferences
- Attended the workshop on "<a href="https://twitter.com/GenebankICRISAT/status/1169137453472669696" target="_blank">A15 DArTSeq Data Analysis</a>" at <a href="https://www.cimmyt.org/" target="_blank">CIMMYT</a>, Mexico
- Attended the international conference on <a href="https://twitter.com/GenebankICRISAT/status/1158297753371308032" target="_blank">"Neglected and Underutilized Crop Species for Food, Nutrition, Energy and Environment"</a> at NIPGR, New Delhi. Awarded a travel grant in recognition of contribution. 
- Attended the workshop on **<a href="https://www.facebook.com/media/set/?vanity=natesan.senthil&set=a.5804435956243670" target="_blank">HPLC: "Principles and Applications in Plant Metabolomics"</a>**" at TNAU,Coimbaotore.
- Attended the workshop on **<a href="https://www.covaichronicle.com/english/contentview/natu?fbclid=IwAR0fA8w7ubx__cGaKnmE-68-tmRLg92XHs1Rf-HnhQzRjmFc_calVar8yFo" target="_blank">Molecular Modelling and Docking</a>** at TNAU, Coimbatore.
- Presented poster titled "**Genes for salt tolerance revealed by functional metagenomics in rice**" at 6<sup>th</sup> National conference AC&RI, TNAU, Trichy.

## ‚úçÔ∏è Articles & Blogs
- <a href="https://medium.com/@victorallan" target="_blank">Medium Articles</a>


    

## üìö Referenser

|     |     |
|:----|:----|
| **Namn:** | **Ass. Prof. Sven Johansson** |
| **Position:** | Head of department, Institution technology and aestethics, DITE, BTH |
| **Organization:** | Blekinge Institute of Technology, Karlskrona, Sweden |
| **E-mail:** | <a href="mailto:sjo@bth.se">email</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/v%C3%A2nia-c-renn%C3%B3-azevedo-21493449/" target="_blank">Visit Page</a> |

|     |     |
|:----|:----|
| **Name:** | **Prof. Mani Vetriventhan** |
| **Position:** | Senior Scientist, Plant Genetic Resources |
| **Organization:** | ICRISAT, Hyderabad, India |
| **E-mail:** | <a href="mailto:M.Vetriventhan@cgiar.org">M.Vetriventhan@cgiar.org</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/vetriventhan-mani-26468322/" target="_blank">Visit Page</a> |

|     |     |
|:----|:----|
| **Name:** | **Mr. Rajaguru Bohar** |
| **Position:** | Regional Genotyping Coordinator (South Asia) / Senior Scientist (Project management) |
| **Organization:** | CIMMYT |
| **E-mail:** | <a href="mailto:wishmeguru@gmail.com">wishmeguru@gmail.com</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/rajaguru-bohar-5038762a/" target="_blank">Visit Page</a> |
|     |     |


## üìû Contact

| **Name**   | Mattias Schertell | 
| **E-mail**   | <a href="mailto:msc@bth.se">arbete</a> | 
| **LinkedIn**   | <a href="https://www.linkedin.com/in/allan-victor-6919b4152/" target="_blank">Visit Page</a> | 
